core/src/main/java/nl/nn/adapterframework/webcontrol/FileViewerServlet.java

```diff
@@ -186,22 +168,15 @@ public static void showReaderContents(Reader reader, String filename, String typ
 				lastPart=filename;
 			}
 			response.setHeader("Content-Disposition","inline; filename=\""+lastPart+"\"");
-			LineNumberReader lnr;
 			if (filename.indexOf("_xml.log")>=0) {
 				Reader fileReader = new EncapsulatingReader(reader, log4j_prefix, log4j_postfix, true);
-				lnr = new LineNumberReader(fileReader);
-			} else {
-				if (filename.indexOf("-stats_")>=0) {
-					Reader fileReader = new EncapsulatingReader(reader, stats_prefix, stats_postfix, true);
-					lnr = new LineNumberReader(fileReader);
-				} else {
-					lnr = new LineNumberReader(reader);
+				try(LineNumberReader lnr = new LineNumberReader(fileReader)) {
```

Comment:
I see no point in using here a `LineNumberReader` instead of a `BufferedReader` since I don't see the line nr being used?

Reply:
Geen zorgen, deze code gaat ook weg. In een volgend pr 😄